src/init.cpp

```diff
@@ -994,6 +995,8 @@ bool AppInit2(boost::thread_group& threadGroup, CScheduler& scheduler)
     if (GetBoolArg("-peerbloomfilters", true))
         nLocalServices |= NODE_BLOOM;
 
+    nMaxTipAge = GetArg("-maxtipage", DEFAULT_MAX_TIP_AGE);
```

Comment:
minor nit, maybe 
```
nMaxTipAge = GetArg("-maxtipage", nMaxTipAge);
``` to not use DEFAULT_MAX_TIP_AGE twice (nMaxTipAge is already initialized to DEFAULT_MAX_TIP_AGE in main.cpp, this seems a little bit redundant)?
Anyway, mino minor minor style nit. Feel more than free to ignore.